plugins/inputs/win_services/win_services.go

```diff
@@ -89,7 +89,7 @@ func (m *winSvcMgr) ListServices() ([]string, error) {
 type mgProvider struct {
 }
 
-func (rmr *mgProvider) Connect() (WinServiceManager, error) {
+func (*mgProvider) Connect() (WinServiceManager, error) {
```

Comment:
Probably also doesn't need to be exported?

Reply:
Many methods in this plugin might be unexported. I will be fixing this as part of the work on the `revive:exported` linter.